teachertool/src/state/actions.ts

```diff
@@ -28,6 +28,11 @@ type SetEvalResult = ActionBase & {
     result: pxt.blocks.EvaluationResult | undefined;
 };
 
+type SetTargetConfig = ActionBase & {
```

Comment:
Is this expected to be hooked up to anything in the state yet?

Reply:
Oops I missed that part. That should have been caught by the linter. Will check settings.

Reply:
The reducer was returning the state unmodified when there was no matching case statement. Removed that.